project_national/sdr_upgrades_tmy3.yml

```diff
@@ -1177,4 +1177,17 @@ upgrades:
   - upgrade_name: HVAC Flexibility - Load Shifting (1hr, 2deg)
     package_apply_logic: *has_cooling_or_heating
     options:
-      - option: HVACFlexibility|pre_only_1hr_2deg
\ No newline at end of file
+      - option: HVACFlexibility|pre_only_1hr_2deg
+
+  - upgrade_name: EV + L2 Charger + DF
+    options:
+      - option: Electric Vehicle Ownership|Yes
+      - option: Electric Vehicle Charger|Level 2 charger
+      - option: EVFlexibility|enabled
+
+  - upgrade_name: Efficienct EV + L2 Charger + DF
+    options:
+      - option: Electric Vehicle Ownership|Yes
+      - option: Electric Vehicle Charger|Level 2 charger
+      - option: Electric Vehicle Efficiency Increase|15%
+      - option: EVFlexibility|enabled
```

Comment:
to-do: apply only to occupied units

Reply:
[b03146c](https://github.com/NREL/resstock/pull/1369/commits/b03146c9c6ea26f85d1ec26c3fc80abf8f6ced38)